### DISCUSSION TITLE
Theme colors for code blocks don't seem to be rendering correctly

### DISCUSSION THREAD

**Discussion by SWHL** [colinwilson/lotusdocs#77] [Q&A] [answered]
Thanks for your support to Prism Theme. I love the theme very much. 

When I change the `prismTheme = twilight`  according to the [docs](https://lotusdocs.dev/docs/guides/features/syntax-highlighting/#themes), the color of code blocks are all the same color and the effect is as follows: ([source](https://github.com/RapidAI/RapidOCRDocs/blob/88d42ec125805e1dad12e531eb2193b0720343e5/content/docs/packages/rapidocr/rapidocr_onnxruntime.md?plain=1#L44))
![image](https://github.com/colinwilson/lotusdocs/assets/28639377/88e6d671-cccf-4ae1-9fbc-b77c35c43f66)

But in the docs, the color is different and the effect is as follows:
![image](https://github.com/colinwilson/lotusdocs/assets/28639377/44e4732c-b81a-4893-8dfa-ff92cfac68f8)

Appreciate any help on this.

**Comment by colinwilson** [comment]
Have you [updated](https://gohugo.io/hugo-modules/use-modules/#update-modules) the Lotus Docs theme?

If not try running `hugo mod get -u` at the root of your project.

**Comment by SWHL** [comment]
I'm sure I am using the latest version of Lotus Docs theme. Otherwise, the background of code blocks using the twilight code theme will not turn black.

I found two 404 pages in my docs page:
![image](https://github.com/colinwilson/lotusdocs/assets/28639377/04574d17-2e6a-4f32-90a8-89defa858ef5)

I guess  the prism theme in the lotus docs theme is not supported the python and bash languages.

**Reply by colinwilson** [reply]
Sorry. `hugo mod get -u`!!!

**Comment by colinwilson** [comment]
Yes. It looks like some of the prism assets are not loading for you.

I can use your example code block without issue:

![2023-09-18 03_16_18-Dark Mode _ Lotus Docs — Mozilla Firefox](https://github.com/colinwilson/lotusdocs/assets/12916656/0e69ef8d-b50a-455a-a20e-ada51b4681b0)

**Comment by SWHL** [comment]
What can I do to fix prism's rendering issues?
Appreciate any help on this.

**Comment by colinwilson** [comment]
It's a path issue. The [PrismJS autoloader plugin](https://prismjs.com/plugins/autoloader/) is looking for `prism-python.min.js` and `prism-bash.min.js` grammars in the wrong place/path:
```
❯ curl -Is https://rapidai.github.io/docs/js/components/prism-python.min.js | grep HTTP
HTTP/2 404
```
The correct path in your case includes `/RapidOCRDocs/`:
```
❯ curl -Is https://rapidai.github.io/RapidOCRDocs/docs/js/components/prism-python.min.js | grep HTTP
HTTP/2 200
```
This is caused by the autoloader language path being hardcoded in the [`lotusdocs/assets/docs/js/prism.js`](https://github.com/colinwilson/lotusdocs/blob/983ce811b0313b571df1e73d5d23957e48b2c692/assets/docs/js/prism.js#L3771) file.

The Prism website notes:
> Note: Autoloader is pretty good at guessing this path. You most likely won't have to change this path. 

I'll remove this in a patch shortly. It should solve your issue once you rebuild your site on GitHub Pages.

**Comment by colinwilson** [comment]
Fixed by a6d0d9b

Update and see if this resolves your issue?

Thanks for pointing this issue out. 👍

**Comment by colinwilson** [comment]
Sorry. The fix doesn't work. I'll update again once I've rectified the issue. Sorry for the noise.

**Comment by colinwilson** [comment, marked as answer]
Fixed by https://github.com/colinwilson/lotusdocs/commit/bc1752e7395a7312711de50effd782fb58ecc225

Run `hugo mod get -u` locally and push to your repo. Should resolve the issue (#78)

**Reply by SWHL** [reply]
I have tried this update, and it indeed fixed the problem. 
Thanks. 
The Lotus Docs theme will be as the main docs theme of my many open-source projects.
